exposed-version-catalog/build.gradle.kts

```diff
@@ -0,0 +1,107 @@
+plugins {
+    `version-catalog`
+    `maven-publish`
+    signing
+}
+
+group = "org.jetbrains.exposed"
+
+catalog {
+    versionCatalog {
+        val version: String by rootProject
+
+        // Core modules
+        library("core", "org.jetbrains.exposed:exposed-core:$version")
+        library("dao", "org.jetbrains.exposed:exposed-dao:$version")
+        library("jdbc", "org.jetbrains.exposed:exposed-jdbc:$version")
+
+        // Extension modules
+        library("java.time", "org.jetbrains.exposed:exposed-java-time:$version")
+        library("jodatime", "org.jetbrains.exposed:exposed-jodatime:$version")
+        library("kotlin.datetime", "org.jetbrains.exposed:exposed-kotlin-datetime:$version")
+        library("json", "org.jetbrains.exposed:exposed-json:$version")
+        library("money", "org.jetbrains.exposed:exposed-money:$version")
+        library("crypt", "org.jetbrains.exposed:exposed-crypt:$version")
+        library("migration", "org.jetbrains.exposed:exposed-migration:$version")
+
+        // Integration modules
+        library("spring.boot.starter", "org.jetbrains.exposed:exposed-spring-boot-starter:$version")
+        library("spring.transaction", "org.jetbrains.exposed:spring-transaction:$version")
+
+        // R2DBC module
+        library("r2dbc", "org.jetbrains.exposed:exposed-r2dbc:$version")
+
+        // BOM
+        library("bom", "org.jetbrains.exposed:exposed-bom:$version")
+
+        // Create bundles
+        bundle("core", listOf("core", "dao", "jdbc"))
+        bundle("datetime", listOf("java.time", "jodatime", "kotlin.datetime"))
+    }
+}
+
+publishing {
+    val version: String by rootProject
+
+    publications {
+        create<MavenPublication>("versionCatalog") {
+            groupId = "org.jetbrains.exposed"
+            artifactId = project.name
+            this.version = version
+            from(components["versionCatalog"])
```

Comment:
@jetbrains-junie, please implement this in buildSrc. Reuse any existing logic if possible.

Reply:
@jetbrains-junie try again to implement this in buildSrc. Reuse any existing logic if possible.